src/server/services/vergunningen.ts

```diff
@@ -263,11 +285,11 @@ export function createVergunningRecentCase(item: Vergunning): MyCase {
 }
 
 export function hasOtherValidVergunningOfSameType(
-  items: ToeristischeVerhuurVergunning[] | GPK[],
-  item: ToeristischeVerhuurVergunning | GPK
+  items: ToeristischeVerhuurVergunning[] | GPK[] | BZP[] | BZB[],
```

Comment:
Is it more clear to use `Array<ToeristischeVerhuurVergunning | GPK | BZP | BZB>` ?